src/main/java/dan200/computercraft/shared/peripheral/generic/data/ItemData.java

```diff
@@ -98,6 +98,12 @@ public static <T extends Map<? super String, Object>> T fill( @Nonnull T data, @
             data.put( "unbreakable", true );
         }
 
+        /*
+         * Execute the detail providers to fill additional data if any.
+         * @see IItemDetailProvider
+         */
```

Comment:
I think we can probably delete these comments. Not sure they add much now that `IItemDetailProvider` (and the like) are gone.